docs/install/knative-with-any-k8s.md

```diff
@@ -444,25 +444,41 @@ To learn more about the Github source, try [our sample](../eventing/samples/gith
 
 {{< /tab >}}
 
-{{% tab name="Camel Source" %}}
-The following command installs the Camel Source:
+{{% tab name="Apache Camel-K Source" %}}
+The following command installs the Apache Camel-K Source:
 
    ```bash
    kubectl apply --filename https://github.com/knative/eventing-contrib/releases/download/{{< version >}}/camel.yaml
    ```
 
-To learn more about the Camel source, try [our sample](../eventing/samples/apache-camel-source/README.md)
+To learn more about the Apache Camel-K source, try [our sample](../eventing/samples/apache-camel-source/README.md)
 
 {{< /tab >}}
 
-{{% tab name="Kafka Source" %}}
-The following command installs the Kafka Source:
+{{% tab name="Apache Kafka Source" %}}
+The following command installs the Apache Kafka Source:
 
    ```bash
    kubectl apply --filename https://github.com/knative/eventing-contrib/releases/download/{{< version >}}/kafka-source.yaml
    ```
 
-To learn more about the Kafka source, try [our sample](../eventing/samples/kafka/source/README.md)
+To learn more about the Apache Kafka source, try [our sample](../eventing/samples/kafka/source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="Apache Kafka Channel" %}}
+The following command installs the Apache Kafka Channel:
+
+   ```bash
+   curl -L "https://github.com/knative/eventing-contrib/releases/download/{{< version >}}/kafka-channel.yaml" \
+    | sed 's/REPLACE_WITH_CLUSTER_URL/my-cluster-kafka-bootstrap.kafka:9092/' \
+    | kubectl apply --filename -
+   ```
+
+Before you can install the Apache Kafka channel, you need to have a running installation of Apache Kafka.
```

Comment:
If you have to do this first, then you should say this first, folks tend not to read thru before doing.

TAL at the Serving Istio instructions for an example of this.  That said, I'd prefer if we could inline a simple guide (Istio has an issue for this).